<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExtractDataDialog</class>
 <widget class="QDialog" name="ExtractDataDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1126</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Extract Data...</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="projectsGroupBox">
     <property name="title">
      <string>Projects</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QListView" name="listView">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QGroupBox" name="variableGroupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0" colspan="2">
       <widget class="QTableView" name="tableView_4">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::MultiSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>200</number>
        </attribute>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="nselvar">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="varSelectAllButton">
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string>Select All</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="varInvertSelectionButton">
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string>Invert Selection</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="varSelectByButton">
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string>Select By...</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="varUnselectButton">
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string>Unselect</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="3" colspan="2">
    <widget class="QGroupBox" name="objectsGroupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="1">
       <widget class="QPushButton" name="objInvertSelectionButton">
        <property name="minimumSize">
         <size>
          <width>131</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string>Invert Selection</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="objSelectByButton">
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string>Select By...</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="objUnselectButton">
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string>Unselect</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="objSelectAllButton">
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string>Select All</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="nselobj">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QTableView" name="tableView_3">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::MultiSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>200</number>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="4" colspan="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>192</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLineEdit" name="dataname"/>
   </item>
   <item row="1" column="2">
    <widget class="QGroupBox" name="dataGroupBox">
     <property name="title">
      <string>Data</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0" colspan="2">
       <widget class="QListView" name="listView_2">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Data Name</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="6">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
